vaadin-components-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/GridElement.java

```diff
@@ -280,10 +280,21 @@ public TestBenchElement getFooterCell(int columnIndex) {
      *            the row to select
      */
     public void select(int rowIndex) {
+        select(getRow(rowIndex));
+    }
+
+    /**
+     * Selects the row.
+     *
+     * @param row
+     *            the row to select
+     */
+    void select(GridTRElement row) {
```

Comment:
Why add this method here instead of in GridTRElement?

Reply:
grid is the one that knows if it's single select
also want to keep `grid.activeItem` workaround for vaadin-grid-flow/74 in a single place

Reply:
Isn’t the API for the user and not for grid? Please choose the one more logical for the user, whichever it is

Reply:
it's not public API

Reply:
the PR is avoiding changes in public API so it can be included in a maintenance release